tips-and-tricks/uv_install.md

```diff
@@ -0,0 +1,60 @@
+# UV Install on HPC
+
+UV is a new Python package manager which offers a few helpful features.
+[Website is here](https://docs.astral.sh/uv/)
+
+This short guide doesn't explain how to use UV generally but just how to get UV to work out of a directory, which is what you would want if you wanted the Python version stored in the project space.
+
```

Comment:
Could we add a line here to mention that root access isn't needed? I think this may be a key phrase people may be looking out for.

```suggestion

This approach is likely to be especially helpful on HPC systems, since no root access, nor an existing Python installation, is required to install it this way.

```